purchase_manual_add_supplierinfo/models/product.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Akretion
```

Comment:
can you rename the file "product.py" to "product_supplierinfo.py" ?
Naming is important

Reply:
@sebastienbeau , it's done.